frontend/src/components/modules/Tournaments/OngoingTournament/PlayoffTournamentView.tsx

```diff
@@ -56,54 +60,59 @@ const PlayoffTournamentView: React.FC = () => {
           justifyContent="flex-start"
           alignItems="flex-start"
         >
-          {Object.entries(rounds).map(([roundNumber, matches], index) => (
-            <React.Fragment key={roundNumber}>
-              {index > 0 && <Divider orientation="vertical" flexItem />}
-              <Grid item>
-                <Box
-                  sx={{
-                    display: "flex",
-                    flexDirection: "column",
-                    alignItems: "center",
-                    minWidth: 300
-                  }}
-                >
-                  <Typography
-                    variant="h6"
+          {Object.entries(rounds).map(([roundNumber, matches], index) => {
+            const roundNrPrint = index + 1;
+            return (
+              <React.Fragment key={roundNumber}>
+                {index > 0 && <Divider orientation="vertical" flexItem />}
+                <Grid item>
+                  <Box
                     sx={{
-                      marginBottom: 2,
-                      textAlign: "center",
-                      textDecoration: "underline"
+                      display: "flex",
+                      flexDirection: "column",
+                      alignItems: "center",
+                      minWidth: 300
                     }}
                   >
-                    {parseInt(roundNumber) === totalRounds
-                      ? t("tournament_view_labels.final")
-                      : `${t("tournament_view_labels.round")} ${roundNumber}`}
-                  </Typography>
-                  {matches.map((match: Match) => {
-                    const tempPlayers: User[] = match.players.map(
-                      (matchPlayer) => {
-                        const player = players.find(
-                          (p) => p.id === matchPlayer.id
-                        );
-                        if (player === null || player === undefined) {
-                          throw new Error();
+                    <Typography
+                      variant="h6"
+                      sx={{
+                        marginBottom: 2,
+                        textAlign: "center",
+                        textDecoration: "underline"
+                      }}
+                    >
+                      {parseInt(roundNumber) === totalRounds
```

Comment:
Jostain syystä tää tarkastelu ei toimi oikein päin. Eli kuvan mukaisesti ekana tulee "finaali" ja sit kierros x.
![image](https://github.com/Kendoers/Kendo-tournament-app/assets/145121433/7d28f7c8-ab2f-41d2-bf26-e7a4a0461eb0)